charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -2072,6 +2072,16 @@ exporters:
         {{- if .Values.presets.loadBalancing.k8s.timeout }}
         timeout: {{ .Values.presets.loadBalancing.k8s.timeout | quote }}
         {{- end }}
+      {{- else if .Values.presets.loadBalancing.awsCloudMap.enabled }}
```

Comment:
Would it make sense to create a new example enabling this? With that, we can validate the resulting config is valid.